fud/fud/stages/verilator/json_to_dat.py

```diff
@@ -111,20 +78,15 @@ def convert2dat(output_dir, data, extension):
     for k, item in data.items():
         path = output_dir / f"{k}.{extension}"
         path.touch()
-        arr = np.array(item["data"])
+        arr = np.array(item["data"], str)
         format = item["format"]
 
-        # Every numeric format shares these two fields.
         numeric_type = format["numeric_type"]
         is_signed = format["is_signed"]
-        shape[k] = {
-            "shape": list(arr.shape),
-            "numeric_type": numeric_type,
-            "is_signed": is_signed,
-        }
+        shape[k] = {"is_signed": is_signed}
 
         if numeric_type not in {"bitnum", "fixed_point"}:
-            raise InvalidNumericType(numeric_type)
+            raise InvalidNumericType("Fud only supports fixed point and bitnum.")
```

Comment:
Is the format string in the JSON file supposed to be "fixed_point" or "fixed point"? It would be useful to indicate the exact string literal expected in the JSON file.